src/views/Streamer.vue

```diff
@@ -239,6 +252,9 @@
       //- )
       router-link#public-link(:to="sessionId" target="_blank")
         | {{ `2n.fm/${sessionId}` }}
+      #copy-button(@click="copyUrl()")
+          CopySvg
+      div#copy-notification.animate-fade-in Copied!
```

Comment:
You don't need `div` in Pug, you can make this line just `#copy-notification.animate-fade-in Copied!`

Reply:
okay! Fixed.